### DISCUSSION TITLE
Add `@updatedAt` attribute to `Schema API` field level

### DISCUSSION THREAD

**Discussion by tsui66** [keystonejs/keystone#6194] [Feature requests]
Automatically stores the time when a record was last updated. If you do not supply a time yourself, the Prisma Client will automatically set the value for fields with this attribute.

**Comment by bladey** [comment]
Hello @tsui66!

A community member has implemented this, check out this thread in Slack - https://keystonejs.slack.com/archives/C01STDMEW3S/p1625725425024200

We may bring this into the core framework at some point, but this should satisfy your needs for now.
